install_config/install/advanced_install.adoc

```diff
@@ -92,6 +92,15 @@ choose an example that matches your requirements, modify it to match your own
 environment, and use it as your inventory file when
 xref:running-the-advanced-installation[running the advanced installation].
 
+[[discrete]]
+[[advanced-install-image-version-policy]]
+=== Image Version Policy
+
+Images require a version number policy in order to maintain updates. See
+the
+xref:../architecture/core_concepts/containers_and_images.adoc#architecture-images-tag-policy][Image
+Version Tag Policy] section in the Architecture Guide for more information.
```

Comment:
This came up in the AsciiDoc guidelines group. Do we format book titles when they are in a sentence: "in the Architecture Guide"?